src/fortran_routing/msh_fortran_CompCh/input/channel_width.txt

```diff
@@ -0,0 +1,501 @@
+0.0 100.0
```

Comment:
Also not a text file etc.

Reply:
.dat? .prn? What's your preference/suggest. It's not a .csv either.